tiling.js

```diff
@@ -2233,6 +2238,10 @@ export const Spaces = class Spaces extends Map {
             monitor.clickOverlay.activate();
         }
 
+        // Update panel to handle target workspace
+        signals.disconnect(Main.panel);
```

Comment:
We don't need to disconnect signals here.  `init` method is called on PaperWM startup - and all signals are destroyed when PaperWM is disabled (including during lockscreen).

Reply:
This disconnect is required because it changes the affected workspace when scrolling. The comment above I do agree with however

Reply:
I see, makes sense.  Could you instead replace it with:

```js
        // Update panel to handle target workspace
        if (this.touchId) {
            signals.disconnect(Main.panel, this.touchId);
        }
        this.touchId = signals.connect(Main.panel, "captured-event", Gestures.horizontalTouchScroll.bind(toSpace));
```
Give the the above test a go.  It's more specific and will disconnect by id.  This way, if we ever want to connect another signal to `Main.panel` it won't also get disconnected.